lib/v2/screens/dashboard/wallet_header.dart

```diff
@@ -2,14 +2,21 @@ import 'package:carousel_slider/carousel_controller.dart';
 import 'package:carousel_slider/carousel_options.dart';
 import 'package:carousel_slider/carousel_slider.dart';
 import 'package:dots_indicator/dots_indicator.dart';
-import 'package:flutter/material.dart';
 import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:seeds/v2/constants/app_colors.dart';
-import 'package:seeds/v2/datasource/remote/model/token_model.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/token_balances_bloc.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/token_balances_event.dart';
+import 'package:seeds/v2/screens/dashboard/interactor/viewmodels/token_balances_state.dart';
 
 import 'components/currency_info_card_widget.dart';
 
 class WalletHeader extends StatefulWidget {
```

Comment:
Why is this page stateful ? 
Generally the only time we needed to have a stateful page so far its been when dealing with forms "User entered input" 

As soon as i see stateful page it smells.
The entire idea behind blocs and states is to move stays away from the UI